app/imports/templates/components/newBid.js

```diff
@@ -18,60 +18,69 @@ Template['components_newBid'].onRendered(function() {
     });
   }
 
-  this.autorun(() => {
-
-    let name = Session.get('searched');
-
-    // The goal here is to obscure the names we actually want
-    function randomName() {
+  // The goal here is to obscure the names we actually want
+  function randomName() {
+    if (typeof knownNames !== "undefined") {
```

Comment:
Can we outright prohibit bids until it's loaded, then? I'd rather not have an invisibly 'weaker' mode of biding.